community-operators/camel-k/camel-k.v0.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,272 @@
+kind: ClusterServiceVersion
+metadata:
+  name: camel-k-operator.v0.1.0
+  namespace: placeholder
+  annotations:
+    categories: "Integration & Delivery"
+    certified: "false" 
+    containerImage: docker.io/apache/camel-k:0.2.0
+    description: |-
+      Apache Camel K (a.k.a. Kamel) is a lightweight integration framework built from Apache Camel that runs natively on Kubernetes and is specifically designed for serverless and microservice architectures.
+    alm-examples: |-
+      [{
+        "apiVersion": "camel.apache.org/v1alpha1",
+        "kind": "Integration",
+        "metadata": {
+          "name": "example"
+        },
+        "spec": {
+          "source": {
+            "content": "import org.apache.camel.builder.RouteBuilder;\n\npublic class Example
+                        extends RouteBuilder {\n  @Override\n  public void configure() throws Exception
+                        {\n\t  from(\"timer:tick\")\n    .setBody(constant(\"-\\n             r\\n             o\\n
+                        \            c\\nHello World\\n             s\\n             !\\n\"))\n\t\t.to(\"log:info?skipBodyLineSeparator=false\");\n
+                        \ }\n}\n",
+            "name": "Example.java"
+          }
+        }
+      }]
+spec:
+  displayName: Camel-K Operator
+  description: |
+    Apache Camel K
+    ==============
+
+    Camel K (a.k.a Kamel) is a lightweight integration framework built from Apache Camel that runs natively on Kubernetes and is specifically designed for serverless and microservice architectures.
+    
+    ## Installation
+
+    To start using Camel K, install the operator and then create the following IntegrationPlatform:
+    ```
+    apiVersion: camel.apache.org/v1alpha1
+    kind: IntegrationPlatform
+    metadata:
+      name: camel-k
+      labels:
+        app: "camel-k"
+    spec:
+      build:
+        camelVersion: "2.23.1"
+        baseImage: "fabric8/s2i-java:3.0-java8"
+        localRepository: "/tmp/artifacts/m2"
+              createdAt: 2019-01-20T12:00:00Z
+              support: Apache Camel Community
+    ```
+    ## Running an Integration
+    
+    After the initial setup, you can run a Camel integration on the cluster by creating an example Integration:
+    ```
+    apiVersion: camel.apache.org/v1alpha1
+    kind: Integration
+    metadata:
+      name: example
+    spec:
+      sources:
+      - content: "import org.apache.camel.builder.RouteBuilder;\n\npublic class Example
+          extends RouteBuilder {\n  @Override\n  public void configure() throws Exception
+          {\n\t  from(\"timer:tick\")\n    .setBody(constant(\"-\\n             r\\n             o\\n
+          \            c\\nHello World!\\n             s\\n             !\\n\"))\n\t\t.to(\"log:info?skipBodyLineSeparator=false\");\n
+          \ }\n}\n"
+        name: Example.java
+    ```
+  keywords: ['apache', 'kamel', 'kubernetes', 'serverless', 'microservices']
+  version: 0.2.0
```

Comment:
This version should match the CSV version. Since the image is tagged `0.2.0` I suggest bumping the CSV to 0.2.0 to match

Reply:
@SamiSousa addressed.